plugins/generator-1.19.4/forge-1.19.4/templates/overlay.java.ftl

```diff
@@ -95,16 +95,32 @@ package ${package}.client.screens;
             </#list>
 
             <#list data.getComponentsOfType("Label") as component>
-                <#assign x = component.x - 213>
-                <#assign y = component.y - 120>
                     <#if hasProcedure(component.displayCondition)>
                         if (<@procedureOBJToConditionCode component.displayCondition/>)
                     </#if>
+                    <#if component.getAnchorPoint().name() == "TOP_LEFT">

```

Comment:
I forgot to mention it, but you should not use the getter method you created inside code templates as it makes tests fail (the method should be used inside the core, to have a general method for all anchorable components). Inside a code template, you need to use the variable you created (so simply `anchorPoint`)

Reply:
LGTM